protos/generate.sh

```diff
@@ -8,68 +8,92 @@ fi
-rm "$pyfile".old
-done
+  echo "Fix imports in generated GRPC service code.."
+  find "$OUT_DIR" -name '*_pb2*.py' | while read -r pyfile; do
+  sed -i'.old' "s/^import \([^ ]*\)_pb2 \(.*\)$/import $PKGNAME.yatai.proto.\1_pb2 \2/" "$pyfile"
+  sed -i'.old' "s/^from \([^ ]*\) import \([^ ]*\)_pb2\(.*\)$/from $PKGNAME.yatai.proto.\1 import \2_pb2\3/" "$pyfile"
+  # Fix google.protobuf package imports
+  sed -i'.old' "s/^import $PKGNAME.yatai.proto.google.\([^ ]*\)_pb2 as \([^ ]*\)$/import google.\1_pb2 as \2/" "$pyfile"
+  sed -i'.old' "s/^from $PKGNAME.yatai.proto.google.\([^ ]*\) import \([^ ]*\)_pb2 as \([^ ]*\)$/from google.\1 import \2_pb2 as \3/" "$pyfile"
+  rm "$pyfile".old
+  done
+}
 
 
-PKG_PATH=$PY_OUT_PATH
-PKGS=$(find "$PKG_PATH" -type d)
-for PKG in $PKGS; do
-    SUBPACKAGES=$(find "$PKG" -maxdepth 1 -type d | grep -E -v "${PKG}$" | sort)
-    MODULES=$(find "$PKG" -maxdepth 1 -iname "*.py" | grep -v "__init__.py" | sort)
-    MODULE_COUNT=$(echo "$MODULES" | wc -w)
-    PKG_INIT="${PKG}/__init__.py"
-    echo "Writing Python package exports"
-    echo "------------------------------"
-    echo "PKG: ${PKG} (${PKG_INIT})"
-    echo "SUBPACKAGES: ${SUBPACKAGES}"
-    echo "FOUND MODULES: $MODULE_COUNT"
-    # echo "MODULES: ${MODULES}"
-    echo ""
-    echo "__all__ = [" > "$PKG_INIT"
-    for MODULE in $MODULES; do
-        FILENAME=$(basename "$MODULE" .py)
-        echo "    \"${FILENAME}\"," >> "$PKG_INIT"
-    done
-    for SUBPKG in $SUBPACKAGES; do
-        SUBPKGNAME=$(basename "$SUBPKG")
-        echo "    \"${SUBPKGNAME}\"," >> "$PKG_INIT"
-    done
-    echo "]" >> "$PKG_INIT"
+fix_grpc_service_code "bentoml" "$PY_OUT_PATH"
+fix_grpc_service_code "tests" "$PY_TEST_OUT_PATH"
+
+
+edit_init() {
+  PKG_PATH=$1
+
+  PKGS=$(find "$PKG_PATH" -type d)
+  for PKG in $PKGS; do
+      SUBPACKAGES=$(find "$PKG" -maxdepth 1 -type d | grep -E -v "${PKG}$" | sort)
+      MODULES=$(find "$PKG" -maxdepth 1 -iname "*.py" | grep -v "__init__.py" | sort)
+      MODULE_COUNT=$(echo "$MODULES" | wc -w)
+      PKG_INIT="${PKG}/__init__.py"
+      echo "Writing Python package exports"
+      echo "------------------------------"
+      echo "PKG: ${PKG} (${PKG_INIT})"
+      echo "SUBPACKAGES: ${SUBPACKAGES}"
+      echo "FOUND MODULES: $MODULE_COUNT"
+      # echo "MODULES: ${MODULES}"
+      echo ""
+      echo "__all__ = [" > "$PKG_INIT"
+      for MODULE in $MODULES; do
+          FILENAME=$(basename "$MODULE" .py)
+          echo "    \"${FILENAME}\"," >> "$PKG_INIT"
+      done
+      for SUBPKG in $SUBPACKAGES; do
+          SUBPKGNAME=$(basename "$SUBPKG")
+          echo "    \"${SUBPKGNAME}\"," >> "$PKG_INIT"
+      done
+      echo "]" >> "$PKG_INIT"
   done
 
-echo "Done"
+  echo "Done"
+}
 
+edit_init "$PY_OUT_PATH"
+edit_init "$PY_TEST_OUT_PATH"
 
+# TODO: if run with generate-docker then we don't have to print this warning
```

Comment:
nit: remove todo, that's seems nice but should be fine

Reply:
noted